schemas/UnauthenticatedError.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "object",
+  "description": "Triggered when the user becomes unauthenticated without actually clicking sign out(E.g. Due to expiry of an access token without a way to refresh).",
+  "properties": {
+    "eventName": {
+      "enum": ["UnauthenticatedError"]
+    },
+    "refreshTokenAuth": {
+      "description": "Whether the auth mechanism is refresh-token-based.",
+      "type": "boolean"
+    },
+    "softLogout": {
+      "description": "Whether a soft logout or hard logout was triggered.",
+      "type": "boolean"
+    },
+    "errorCode": {
+      "description": "The error code as defined in matrix spec.",
```

Comment:
what values can this be?

Reply:
Ideally it'd be an enum with the known values, but if you're not controlling it, fair enough.

Can you add that context to the description please?

Reply:
Actually now that I think of it we only force the logout and call analytics event on particular server response so I can provide an enum for this.